pkg/services/ngalert/api/tooling/definitions/provisioning_alert_rules.go

```diff
@@ -192,7 +192,7 @@ type ProvisionedAlertRule struct {
 
 // swagger:route PUT /v1/provisioning/folder/{FolderUID}/rule-groups/{Group} provisioning stable RoutePutAlertRuleGroup
 //
-// Update the interval of a rule group.
+// Create or update alert rule group.
```

Comment:
I think we need to regenerate the openAPI docs for this? This text ends up in the openapi doc and is generally misleading, i think it would be nice to get it out of there.